sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ProtoDescriptorRenderer.scala

```diff
@@ -32,7 +32,10 @@ object ProtoDescriptorRenderer {
       builder ++= " {\n"
       messageType.getFieldList.forEach { field =>
         builder ++= "  "
-        builder ++= field.getTypeName
+        if (field.hasTypeName)
+          builder ++= field.getTypeName
+        else
+          builder ++= field.getType.name().toLowerCase.drop(5)
```

Comment:
Just curious, but why this? And what is this drop 5?

Reply:
Primitive type name is always prefixed by `TYPE_`, this drops that prefix.